### PR TITLE
First working version

### PR DESCRIPTION
**Pull request by maciekrb**
MALOS `asyncio` micro lib
Implements coroutines that handle MALOS keep-alive, error and data ports in an `asyncio` loop. 

Includes a simple CLI for connecting and reading data from the MATRIX Creator and basic getting started docs.

Packaged as a python package that will be available on pypi.

### CODE REVIEW

File: matrix_io/malos/drivers.py [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+    # updates. Pings are useful to prevent blocked applications from keeping
+    # a MALOS driver bussy.
+    while True:
+        try:
+            # Ping with empty string to let the drive know we're still listening
+            await sock.send_string('')
+
+            # Delay between next ping
+            await asyncio.sleep(delay)
+        except asyncio.CancelledError:
```
Comment (arhuaco) [inline matrix_io/malos/drivers.py]: Does it make sense to print other exceptions? Even if you don't die on them.

**Review by arhuaco — COMMENTED** [review]
Commenting. Looks nice.

File: matrix_io/malos/drivers.py (hunk 1 above)
Comment (maciekrb) [reply matrix_io/malos/drivers.py]: In this case any other exception will propagate. We are catching the `CancelledError`  as in this case we just want to exit the `while True`  loop gracefully. References here https://docs.python.org/3/library/asyncio-task.html

Comment (maciekrb) [reply matrix_io/malos/drivers.py]: However, when using `ensure_future`, exceptions will not crash the system and might go unnoticed, so I will try to work around this to make the code easier to debug.

**Review by hpsaturn — APPROVED** [review]
Only one thing, you should add some example maybe? with this could we change the examples in matrix-creator-malos to this schema?

**Comment by maciekrb** [comment]
Sure @hpsaturn. I added 2 simple examples, one presenting an example using the command line client (which you can use directly to test) and the other one  using the `MalosLoop` class. What other examples would you like to see here ?

**Comment by hpsaturn** [comment]
@maciekrb this samples are ok, maybe the next step is change the MALOS samples? maybe do you it? or migrate the old samples to here?

**Comment by heitorgo1** [comment]
Maybe it's time to merge this to master, what do you guys think?

**Review by heitorgo1 — APPROVED** [review]

**Pull request merged**